pyproject.toml

```diff
@@ -172,9 +179,32 @@ ignore = [
 ]
 
 [tool.pytest.ini_options]
+markers = [
+    "requires_jupytext",
+    "requires_black",
+    "requires_isort",
+    "requires_flake8",
+    "requires_autopep8",
+    "requires_nbconvert",
+    "requires_myst",
+    "requires_no_myst",
+    "requires_quarto",
+    "requires_pandoc",
+    "requires_no_pandoc",
+    "requires_sphinx_gallery",
+    "requires_user_kernel_python3",
+    "requires_ir_kernel",
+    "skip_on_windows",
+    "pre_commit",
+]
 filterwarnings = [
     # Uncomment this "error" to turn all unfiltered warnings into errors
     # "error",
+    # Jupyter notebook
+    "ignore:Support for bleach <5 will be removed in a future version of nbconvert:DeprecationWarning",
+    # jupyterfs
+    "ignore:run_pre_save_hook is deprecated, use run_pre_save_hooks instead:DeprecationWarning",
+    "ignore:Deprecated call to `pkg_resources.declare_namespace:DeprecationWarning",
```

Comment:
You can make a new issue from this comment.

Are these warnings triggered automatically at module import or at function call. If the latter, it should be investigated how to avoid them. E.g. `pkg_resources` might be replaced with `importlib`.

Reply:
Yes - but that issue is with an old version of `jupyter-fs` so I'm not sure it's worth reporting it